texthero/representation.py

```diff
@@ -136,42 +48,38 @@ def count(
     min_df=1,
     max_df=1.0,
     binary=False,
-) -> pd.Series:
+) -> pd.DataFrame:
     """
     Represent a text-based Pandas Series using count.
 
-    Return a Document Representation Series with the
-    number of occurences of a document's words for every
-    document.
+    Rows of the returned DataFrame represent document whereas 
```

Comment:
"represent document**s**"

change also in `term_frequency` + `tfidf`

Reply:
:heavy_check_mark: